src/components/modal/addtostudio/studio-button.jsx

```diff
@@ -10,19 +11,33 @@ const StudioButton = ({
     id,
     includesProject,
     title,
-    onToggleStudio
+    onClick,
+    wasClicked
 }) => {
     const checkmark = (
         <img
             alt="checkmark-icon"
-            className="studio-status-icon-checkmark-img"
+            className={classNames(
+                'studio-status-icon-checkmark-img',
+                {'studio-status-icon-with-animation': wasClicked}
+            )}
             src="/svgs/modal/confirm.svg"
         />
     );
+    const spinner = (
+        <img
+            alt="loading animation"
+            className="studio-status-icon-spinner"
+            src="/svgs/modal/spinner.svg"
+        />
+    );
```

Comment:
Ohhh... hm the issue said I should use the svg so that's what I did? The spinner component would have to be changed if it started using the svg, is that what we want?

Reply:
I'm not sure. IMO we should avoid having different styles of spinners unless there's a good reason, so I think it'd be acceptable to update the spinner component to use this new style if we want to change it. I think @carljbowman or @kathymakes should weigh in though!

Reply:
I agree that we should use the same spinner component everywhere. 

That said, I think it would be worth considering updating that base spinner component. If my memory serves me correctly, we grabbed some random spinner for sake of ease. It'd be good to be more intentional as this type of feedback becomes more apart of our UI feedback for API calls.

@kathymakes was also exploring using a spinner on the Editor Message system for things like saving.

Reply:
Cool so I will go and update the spinner component and use that then! Should I make a whole new issue for this or just keep working here?

Reply:
I think it would make sense to make a new pull request for just the spinner component @rschamp